test.sh

```diff
@@ -130,7 +159,7 @@ function nightly_functionality_stress_tests() {
     local dbname="splinter_test.functionality.db"
     echo "$Me: Run ${test_name} with ${n_mills} million rows, on ${ntables} tables, with ${cache_size} GiB cache"
     run_with_timing "Functionality Stress test ${test_descr}" \
-            $BINDIR/driver_test splinter_test --functionality  ${num_rows} 1000 \
+            "$BINDIR"/driver_test splinter_test --functionality  ${num_rows} 1000 \
```

Comment:
These changes from `$BINDIR` to `"$BINDIR"` are cosmetic?

Reply:
Not exactly. Otherwise we get errors from `shellcheck` and that step will fail.

Reply:
Ack